tests/Wpf/Prism.Wpf.Tests/Prism.Wpf.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
   <PropertyGroup>
-    <TargetFramework>net461</TargetFramework>
+    <TargetFramework>net472</TargetFramework>
```

Comment:
don't you want to update this like you did the Prism.Core.Tests to multi-target?

Reply:
not in this PR, that will be a separate effort